azurerm/provider.go

```diff
@@ -119,6 +119,7 @@ func Provider() terraform.ResourceProvider {
 		"azurerm_recovery_services_vault":                 dataSourceArmRecoveryServicesVault(),
 		"azurerm_recovery_services_protection_policy_vm":  dataSourceArmRecoveryServicesProtectionPolicyVm(),
 		"azurerm_redis_cache":                             dataSourceArmRedisCache(),
+		"azurerm_resource":                                dataSourceArmResource(),
```

Comment:
since this is returning multiple resources could we make this:

```suggestion
		"azurerm_resources":                                dataSourceArmResources(),
```